portal/src/main/webapp/WEB-INF/jsp/global/global_variables.jsp

```diff
@@ -172,19 +173,15 @@
         var converted_oql = oql_html_conversion_vessel.textContent.trim();
         window.QuerySession = window.initDatamanager('<%=geneticProfiles%>'.trim().split(/\s+/),
                                                             converted_oql,
-                                                            ['<%=cancerTypeId%>'.trim()],
-                                                            '<%=samples%>'.trim().split(/\s+/),
+                                                            ['<%=cancerStudyId%>'.trim()],
```

Comment:
That's a good point. @jjgao would there ever be a case where a study was selected without any explicit samples?